.github/workflows/pr.yaml

```diff
@@ -318,7 +318,7 @@ jobs:
   unit-tests-cudf-pandas:
     needs: [wheel-build-cudf, changed-files]
     secrets: inherit
-    uses: rapidsai/shared-workflows/.github/workflows/wheels-test.yaml@branch-25.06
+    uses: rapidsai/shared-workflows/.github/workflows/wheels-test.yaml@cuda-12.9.0
```

Comment:
Noticed this workflow was newly added and is using `branch-25.06`. Have changed it to `cuda-12.9.0`